lib/Importer/ONNXModelLoader.cpp

```diff
@@ -180,6 +180,21 @@ static void loadTensor(const ONNX_NAMESPACE::TensorProto &in, Tensor *T) {
     } else {
       llvm_unreachable("Unsupported Tensor format.");
     }
+  } else if (in.data_type() == ONNX_NAMESPACE::TensorProto::INT32) {
```

Comment:
Can you please add a few lines of comments that explain what's going on?

Reply:
Ok.  Its just one extra possible data type that we need to support. The code here is highly similar.

Reply:
Ok.

Reply:
I added the comments below this line on where int32 tensor comes from.

Reply:
@artemrakhov The code around here in other if statements look undocumented. Can we go over and add a few comments that explain what we need the i-stream and other stuff that are not trivial?

Reply:
Ah, because the data we want can lie in either field like `int32_data`, `int64_data`, or it can be packed in the `raw_data` string.